src/js/tech/html5.js

```diff
@@ -99,7 +99,7 @@ class Html5 extends Tech {
 
     if (this.featuresNativeTextTracks) {
       if (crossoriginTracks) {
-        log.warn(tsml`Text Tracks are being loaded from another origin but the crossorigin attribute isn't used. 
+        log.warn(tsml`Text Tracks are being loaded from another origin but the crossorigin attribute isn't used.
```

Comment:
`tsml` will strip spaces between lines so this will end up with "isn't used.This may", which isn't the end of the world, but we could use my alternative package `tsmlj` which will join each line with a space.